cmd/fyne/commands/deprecated.go

```diff
@@ -25,21 +25,19 @@ func Vendor() *cli.Command {
 	}
 }
 
-// Version returns the cli command for the program version
+// Version returns the cli command for the program version.
 //
-// Deprecated: Use "go mod vendor" instead
+// Deprecated: Use "fyne --version" or "fyne -v" instead.
 func Version() *cli.Command {
 	return &cli.Command{
 		Name:  "version",
 		Usage: "Deprecated: Use \"fyne --version\" or \"fyne -v\" instead.",
 		Action: func(_ *cli.Context) error {
-			if info, ok := debug.ReadBuildInfo(); !ok {
-				return errors.New("could not retrieve version information (ensure module support is activated and build again)")
-			} else {
+			if info, ok := debug.ReadBuildInfo(); ok {
 				fmt.Println("fyne cli version:", info.Main.Version)
 			}
 
-			return nil
+			return errors.New("could not retrieve version information (ensure module support is activated and build again)")
```

Comment:
Wouldn't this error be returned even if a `fyne cli version` was successfully returned?

Reply:
Good point. I had missed a return.